Copy edits on git day 1 initial workshop (#93)
CodeYourFuture/CYF-Workshops: git-day-1/README.md

![vscode-sidebar-source-control](vscode-sidebar-source-control.png)

2. Go to the Changes section and click on the file you changed - this should now show the changes for the file.
3. Try editing the file again in the Explorer tab and check to see the update is visible in the Source Control panel

### 🟢 3. Stage the changes

We need to tell Git which changes we want to be part of our next commit.

Each commit is a checkpoint we've decided to save. When making a commit, we can decide to _not_ include all of our changes in the checkpoint, but just include some of them.

> We choose which changes we do want to include in the commit by **staging our changes**.

Reviewer: ```suggestion
We choose which changes we want to include in a commit by **staging our changes**.
```

In the Source Control tab again...

1. Go to the file `README.md` and click on the **+**.

👓 Notice what happens when you carry out step 1.

> (In this course, you will probably want to just stage all of your changes most of the time).

Reviewer: I don't think we need to say this - I fear them using `git add .` in the future
